service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/RemoveOrderControllerAboutToSubmitTest.java

```diff
@@ -258,6 +263,50 @@ void shouldRemoveSDONoticeOfProceedingsAndSetStateToGatekeepingWhenRemovingASeal
         assertNull(responseData.getNoticeOfProceedingsBundle());
     }
 
+    @Test
+    void shouldRemoveDraftCaseManagementOrderAndRemoveHearingAssociation() {
+        UUID removedOrderId = UUID.randomUUID();
+        UUID additionalOrderId = UUID.randomUUID();
+        UUID hearingOrderBundleId = UUID.randomUUID();
+
+        Element<HearingOrder> caseManagementOrder1 = element(removedOrderId, HearingOrder.builder()
+            .status(DRAFT)
+            .type(HearingOrderType.DRAFT_CMO)
+            .build());
+
+        List<Element<HearingOrder>> ordersToBeRemoved = newArrayList(
```

Comment:
name is a bit misleading. It contains two orders but only one of these we actually remove. If that was response to @toireasam comment I think what she meant was to name caseManagementOrder1 - orderToBeRemoved.